build-scripts/RTBuilder_32b

```diff
@@ -33,25 +33,36 @@ helpfunction() {
 	figlet -t -k -f /usr/share/figlet/term.flf "Copyright (c) 2020 abhiTronix"
 	echo ""
 	echo ""
-	echo "Usage: $0 -g [GCC version] -r [Target Pi type] -o [Target Pi OS type]"
+	echo "Usage: $0 -g [GCC version] -r [Target Pi type] -o [Target Pi OS type] -V"
 	echo -e "\t-g GCC version you want to compile?: (7.1.0|7.2.0|7.3.0|7.4.0|7.5.0|8.1.0|8.2.0|8.3.0|9.1.0|9.2.0|9.3.0|10.1.0|10.2.0)"
 	echo -e "\t-r What's yours Target Raspberry Pi type?: (0-1|2-3|3+)"
 	echo -e "\t-o What's yours Target Raspberry Pi OS type?: (stretch|buster)"
+	echo -e "\t-V Verbose output for debugging?"
 	echo ""
 	echo ""
 	exit 1 # Exit script after printing help
 }
 
 #input arguments handler
-while getopts "g:r:o:" opt; do
+while getopts "g:r:o:V" opt; do
 	case "$opt" in
 	g) GCC_VERSION="$OPTARG" ;;
 	r) RPI_TYPE="$OPTARG" ;;
 	o) RPIOS_TYPE="$OPTARG" ;;
+	V)
+		VERBOSE=1
+		echo "Info: Activated verbose output!"
+		echo ""
+		set -eov pipefail # Set verbose mode.
```

Comment:
Hm. The script will fail on error only if someone chooses the verbose option. Errors would still be silently ignored otherwise.

Wouldn't it make sense to put `set -eo pipefail` at the top of every script and call `set -v` during cmdline parsing in scripts which support the verbose option?